File: Course_01/C Language/Vetores/aprovados.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n;

    printf("Quantos alunos serao digitados? ");
    scanf("%d", &n);

    char nomes[n][50];
    double n1[n], n2[n], media;

    for(int i = 0; i < n; i++) {
        printf("Digite nome, primeira nota e segunda nota do %do aluno:\n", i + 1);
        fseek(stdin, 0, SEEK_END);
        fgets(nomes[i], 50, stdin);
        scanf("%lf", &n1[i]);
        scanf("%lf", &n2[i]);
    }

    printf("Alunos aprovados:\n");

    for(int i = 0; i < n; i++) {
        media = (n1[i] + n2[i]) / 2;
        if (media >= 6) {
            printf("%s", nomes[i]);
        }
    }

    return 0;
}

File: Course_01/Python/Vetores/aprovados.py
n = int(input("Quantos alunos serão digitados? "))

nome = [0 for x in range(n)]
media = [0 for x in range(n)]

for i in range(0, n):
    print(f"Digite nome, primeira e segunda nota do {i + 1}o aluno:")
    nome[i] = input()
    n1 = float(input())
    n2 = float(input())
    media[i] = (n1 + n2) / 2

print("Alunos aprovados:")
for i in range(0, n):
    if media[i] >= 6:
        print(nome[i])